recipes/Cfg/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cfg" %}
+{% set version = "9.0.0" %}
+{% set tag = "v9r0" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: https://git.ligo.org/virgo/virgoapp/Cfg/-/archive/{{ tag }}/Cgf-{{ tag }}.tar.gz
+   sha256: 40ff7b98d515e754a2d6d0a2eebdd995483a63a4fa7c3b804ea79c7b9ca5bb24
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - meson
+    - ninja
+    - make  # [unix]
+    - pkg-config  # [unix]
+  host:
+    - cm >=9.0.0
+  run:
+    - cm >=9.0.0
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - test -r ${PREFIX}/lib/libcfg.so
+    - test "$(pkg-config --print-errors --modversion cfg)" == "${PKG_VERSION}"  # [unix]
```

I think you copied this from the `framel` feedstock, but since I wrote that I figured out a much more elegant version:

```suggestion
    - pkg-config --print-errors --exact-version ${PKG_VERSION} ${PKG_NAME}  # [unix]
```